Sorts/test/PatienceSort.test.js

```diff
@@ -0,0 +1,25 @@
+import { patienceSort } from '../PatienceSort'
+
+test('The PatienceSort of the array [5, 4, 3, 2, 1] is [1, 2, 3, 4, 5]', () => {

```

Comment:
Please use `each`: https://github.com/TheAlgorithms/TypeScript/pull/42#pullrequestreview-1135228816
(side note: eventually the tests for all sorting algorithms should be consolidated as I have done in the Lua repo)